conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "meta-trenchboot-demo"
+BBFILE_PATTERN_meta-trenchboot-demo = "^${LAYERDIR}/"
+BBFILE_PRIORITY_meta-trenchboot-demo = "6"
+
+LAYERDEPENDS_meta-trenchboot-demo = "core openembedded-layer"
```

Comment:
@Asiderr This layer depends on meta-trehncboot, meta-pcengines etc. It should be reflected here

Reply:
@macpijan Fixed in the last commit.